src/commandSocket.cpp

```diff
@@ -158,9 +167,9 @@ void CommandSocket::connect(const std::string& ip, int port)
             //sendPrintTime();
         }
 
-        if(!private_data->socket->errorString().empty()) 
+        if(private_data->socket->getLastError().isValid())
         {
-            logError("%s\n", private_data->socket->errorString().data());
+            logError("%s\n", private_data->socket->getLastError().toString());
```

Comment:
The Arcus error is converted to string in the following format:

`Arcus <Fatal?> Error (<ErrorCode>): <Error Description>`

If that is not sufficient I can add the socket error bit.